### DISCUSSION TITLE
Bagaimanakah menentukan nilai frekuensi utk data Time Series yg harian menjadi Bulanan ?

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#607] [Q&A] [answered]
Jika kita memiliki data set Time Series harian dan ingin melihatnya secara bulanan, bagaimanakah  menentukan nilai frekuensinya sementara jumlah hari dalam setiap bulannya bisa berbeda-beda ? (Contoh data time series Omset harian selama beberapa tahun yg ingin diamati secara bulanan)

Terimakasih

**Comment by Yusraf002** [comment, marked as answer]
Dear @Micoalgo 

Terima kasih atas pertanyaannya Pak, 
untuk contoh kasus Omset harian yang akan diamati pola bulanan kita hanya perlu menggunakan nilai frekuensi sebesar **30**. 

Demikian yang dapat kami sampaikan. Semoga membantu!

Best regards,
Team Algoritma

**Reply by Micoalgo** [reply]
Teerimakasih utk jawabannya mas, kalau misalnya kita ingin cut off perbulannya scr tepat misal : Februari 28 Hari, Maret 31 Hari dan ada bulan2 lainnya yg tidak 30 hari, bagaimana ya ? (karena forecasting yg akan dilakukan rencananya per bulan, sehingga insight yg ingin diambil juga dari pengamatan per bulan tsb)

**Comment by Yusraf002** [comment]
Terimakasih untuk tanggapannya pak @Micoalgo ,
Untuk hal tersebut di r tidak bisa diakukan pak, dan memang idealnya untuk data harian dan ingin dilihat pola perbulan akan menggunakan general 30.

**Comment by Micoalgo** [comment]
Ok mas, Terimakasih jawabannya

**Comment by jafarhf1** [comment]
Dear Pak @Micoalgo,
Berikut kami lampirkan referensi yang juga bisa dijadikan pertimbangan terkait pertanyaan Bapak:
https://rpubs.com/Mentors_Ubiqum/Time_Series_2

Best Regards,
Team Algoritma

**Comment by Micoalgo** [comment]
Terimakasih Mas Jafar
